### ISSUE TITLE
Containers running on new Docker WSL2 Backend throwing error when trying to attach VS Code

### ISSUE THREAD

**Issue by fadlio** [microsoft/vscode-remote-release#1782]
Issue Type: <b>Bug</b>

After the newest Docker Desktop update (2019-11-05), which replaced WSL2 Tech Preview for WSL2 Backend, I'm no longer able to attach VS Code to containers.

<b>Expected result:</b>
Using Docker extension, from the container listing, attach VS Code to the container.

<b>Actual result:</b>
When trying to attach to the container in the containers list, VS throw the following error: "Docker returned an error. Make sure the Docker daemon is running."

Docker Desktop version: 2.1.5.0 (40323)

Extension version: 0.83.0
VS Code version: Code - Insiders 1.40.0-insider (d39e6b396c3ea364584126316a0fcf30f9cb378e, 2019-11-05T02:01:04.347Z)
OS version: Windows_NT x64 10.0.19013

![image](https://user-images.githubusercontent.com/6645818/68176069-2a1e0880-ff62-11e9-8449-789fabbe0078.png)

**Comment by Chuxel** [comment]
@fadlio A few things here with 2.1.5.0 and updates to the Docker extension.

1. I noticed you were using the docker extension in the screen shot above.  Does it work if you use the **Remote Explorer** to attach instead?  I am able to successfully attach using this method.
2. Did you have VS Code connected to WSL at the time?  The Docker extension recently updated to a `workspace` extension which breaks its ability to use this command (see #1826).  We will be removing it in this situation favor of the remote explorer which will never have this problem.
3. One thing I noticed about this update is that the change actually seemed to break the WSL context since they now update the default context.  Did you run `docker context use default`?
